features/admin.users.v1/components/user-profile.tsx

```diff
@@ -1282,7 +1358,560 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
+        if (!isEmpty(primaryAttributeValue)) {
+            attributeValueList = attributeValueList.filter((value: string) =>
+                !isEmpty(value)
+                && value !== primaryAttributeValue);
+            attributeValueList.unshift(primaryAttributeValue);
+        }
+        const showAccordion: boolean = attributeValueList.length >= 1;
+        const accordionLabelValue: string = showAccordion ? attributeValueList[0] : "";
+
+        const showVerifiedPopup = (value: string): boolean => {
+            return verificationEnabled && verifiedAttributeValueList.includes(value);
+        };
+
+        const showPrimaryPopup = (value: string): boolean => {
+            return value === primaryAttributeValue;
+        };
+
+        const showMakePrimaryButton = (value: string): boolean => {
+            if (verificationEnabled) {
+                return verifiedAttributeValueList.includes(value) && value !== primaryAttributeValue;
+            } else {
+                return value !== primaryAttributeValue;
+            }
+        };
+
+        const showVerifyButton = (value: string): boolean =>
+            schema.name === ProfileConstants.SCIM2_SCHEMA_DICTIONARY.get("EMAIL_ADDRESSES")
+            && verificationEnabled
+            && !verifiedAttributeValueList.includes(value);
+
+        return (
+            <>
+                <Field
+                    action={ { icon: "plus", type: "submit" } }
+                    data-testid={ `${ testId }-profile-form-${ schema.name }-input` }
+                    name={ schema.name }
+                    label={ schema.name === "profileUrl" ? "Profile Image URL" :
+                        (  (!commonConfig.userEditSection.showEmail && schema.name === "userName")
+                            ? fieldName +" (Email)"
+                            : fieldName
+                        )
+                    }
+                    required={ schema.required }
+                    requiredErrorMessage={ fieldName + " " + "is required" }
+                    placeholder={ "Enter your" + " " + fieldName }
+                    type="text"
+                    key={ key }
+                    disabled={ schema.name === "userName" }
+                    readOnly={ isReadOnly || schema.mutability === ProfileConstants.READONLY_SCHEMA }
+                    validation={ (value: string, validation: Validation) => {
+                        if (!RegExp(schema.regEx).test(value)) {
+                            validation.isValid = false;
+                            validation.errorMessages
+                                .push(t("users:forms.validation.formatError", {
+                                    field: fieldName
+                                }));
+                        }
+                    } }
+                    maxLength={
+                        fieldName.toLowerCase().includes("uri") || fieldName.toLowerCase().includes("url")
+                            ? 1024
```

Comment:
include this value in a const file

Reply:
Introduced a new variable in ProfileConstants.
https://github.com/wso2/identity-apps/pull/6563/commits/3c27b8f9c21c0d65f02235306cfe3d1815171a92